templates/project/.github/workflows/test.yaml.twig

```diff
@@ -82,6 +82,10 @@ jobs:
             - name: Add PHPUnit matcher
               run: echo "::add-matcher::{% verbatim %}${{ runner.tool_cache }}{% endverbatim %}/phpunit.json"
 
+            - name: Use older Flex endpoint for Symfony 4.4
+              if: matrix.symfony-require == '4.4.*'
+              run: composer config extra.symfony.endpoint https://raw.githubusercontent.com/symfony/recipes/c8d90298241f176a261dadc4814d7058415a4c1f/index.json
```

Comment:
Should we add this conditionally to the fact we have symfony 4.4 in the variants ?
This way, the more we remove 4.4 support the more we won't have this in the yaml file

Reply:
Should be good now.